ui/common/build.gradle.kts

```diff
@@ -14,6 +17,22 @@ kotlin {
                 implementation(projects.data.network)
                 api(libs.com.arkivanov.decompose)
                 api(libs.com.arkivanov.decompose.extensions.compose.jetbrains)
+                implementation("com.alialbaali.kamel:kamel-image:0.4.0")
+                implementation("it.skrape:skrapeit:1.2.2")
```

Comment:
I think the declaration of dependencies is missing in the `libs.versions.tom` file.

Reply:
thanks fixed